hosting/.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "semi": true,
+  "singleQuote": true,
```

Comment:
It changed all the code which was already double quote 😭 
Most examples I find on the web and [vercel examples](https://github.com/vercel/next.js/blob/canary/examples/api-routes-graphql/pages/index.tsx) are using double quotes. Let's follow that for easier standardization.